workspaces/desktop/pre-install.sh

```diff
@@ -4,7 +4,7 @@ set -ex
 apt-get update && apt-get upgrade -y
 
 # bring back the missing manpages
-apt-get install -y man-db
+apt install -y man-db
```

Comment:
It might not make a big difference, but the `apt` documentation still recommends using `apt-get` in scripted contexts, because the API for the more user-friendly `apt` tool is subject to change: https://manpages.ubuntu.com/manpages/jammy/en/man8/apt.8.html#script%20usage%20and%20differences%20from%20other%20apt%20tools

Reply:
Replaced in 2170671101dc6f6169d771d28fac120943048445.